recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
Kyle mentions that it could be that the build environment somehow leaks into the test environment. Even if it magically works, I think we should try to adhere to the FHS. Hence, executables should go to $PREFIX/bin.